packages/nimbus/src/components/toggle-button-group/toggle-button-group.mdx

```diff
@@ -12,223 +12,442 @@ menu:
+> -   You may amend a "more" button at the end that is icon only to organize more options if needed. Otherwise it is not good practice to mix styles in button groupings.
 
 ```jsx-live
 const App = () => (
-  <Stack direction="column" gap="400">
-    <Text>Single Selection with Deselection Allowed (default)</Text>
-    <ToggleButtonGroup.Root
-      aria-label="Default selection behavior"
-      selectionMode="single"
-    >
-      <ToggleButtonGroup.Button id="single-left">Left</ToggleButtonGroup.Button>
-      <ToggleButtonGroup.Button id="single-center">Center</ToggleButtonGroup.Button>
-      <ToggleButtonGroup.Button id="single-right">Right</ToggleButtonGroup.Button>
-    </ToggleButtonGroup.Root>
+  <ToggleButtonGroup.Root aria-label="Filter status" selectedKeys={["filter-accepted"]} tone="primary">
+    <ToggleButtonGroup.Button id="filter-drafts">
+      Drafts (6)
+    </ToggleButtonGroup.Button>
+    <ToggleButtonGroup.Button id="filter-accepted">
+      Accepted (2)
+    </ToggleButtonGroup.Button>
+    <ToggleButtonGroup.Button id="filter-rejected">
+      Rejected (10)
+    </ToggleButtonGroup.Button>
+  </ToggleButtonGroup.Root>
+);
+```
 
-    <Text>Multiple Selection</Text>
-    <ToggleButtonGroup.Root
-      aria-label="Multiple selection example"
-      selectionMode="multiple"
-    >
-      <ToggleButtonGroup.Button id="multiple-bold">Bold</ToggleButtonGroup.Button>
-      <ToggleButtonGroup.Button id="multiple-italic">Italic</ToggleButtonGroup.Button>
-      <ToggleButtonGroup.Button id="multiple-underline">Underline</ToggleButtonGroup.Button>
-    </ToggleButtonGroup.Root>
-  </Stack>
+> [!CAUTION]\
+> **Don't**
+>
+> -   Do not confuse the user with unclear options.
+> -   Do not use unfamiliar and lengthly labels.
+
+```jsx-live
+const App = () => (
+  <ToggleButtonGroup.Root aria-label="Filter status (Bad Example)" selectedKeys={["filter-granted"]} tone="primary">
+    <ToggleButtonGroup.Button id="filter-compositions">
+      Compositions
+    </ToggleButtonGroup.Button>
+    <ToggleButtonGroup.Button id="filter-granted">
+      Granted
+    </ToggleButtonGroup.Button>
+    <ToggleButtonGroup.Button id="filter-veto">
+      Veto
+    </ToggleButtonGroup.Button>
+  </ToggleButtonGroup.Root>
 );
 ```
 
-## Controlled Mode
+### Offering many options
+
+In toolbars, button groups can organize related tools or actions.
 
-You can control the selection state externally:
+> [!TIP]\
+> **Do**
+>
+> -   Use expected tooling and order that a user will be familiar with when associated with effected content, like a rich-text editor.
+> -   Use consistent styling when creating toolbars or curated sets of tools for the user. Include tooltips to label icons.
+> -   The last button can be paired with a popover to feature additional options. Use the vertical three dot "More" icon for consistency throughout the UI.
```

Comment:
@misama-ct @ByronDWall @FilPob I mentioned this here in [this thread in Slack](https://commercetools.slack.com/archives/C045CH797B2/p1745605928323629), but just repeating here for consistency - I think this pattern of having a "last button that does something different than the rest" is strange, and I don't know that consumers will clearly understand how to implement this 🤔

Reply:
The 3-vertical-dots 'see more' menu is pretty standard, I personally think that we should allow for it in order to reduce visual clutter and maintain a visual hierarchy for important vs less important actions

That being said, my care amount here is v low - whatever we decide is fine as long as it's implemented consistently

Reply:
Removed in 2a11db101a576a62e447287aed3144e4b589c9a1